app/models/json_schemas/imported_from_feed.json

```diff
@@ -0,0 +1,15 @@
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"title": "Transitland Datastore Stop",
+	"type": "object",
+	"required": ["feedVersion"],
+	"additionalProperties": false,
+	"properties": {
+		"feedVersion": {
+			"type": "string"
+		},
+		"gtfsId": {
+			"type": ["string", "null"]
+		}
```

Comment:
Also allow feed Onestop ID as input? Or you think it's better to just rely on the uniqueness of feed version SHA1s?